app/views/RoomView/index.tsx

```diff
@@ -607,13 +606,11 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 
 			this.setState({ canAutoTranslate, member, loading: false });
 		} catch (e) {
+			log(e);
```

Comment:
This logic is meant to fail when there's connectivity issues.
We shouldn't log this, because it's only going to flood our analytics.